Sources/StripeKit/Core Resources/PaymentIntents/PaymentIntent.swift

```diff
@@ -76,11 +76,18 @@ public struct StripePaymentIntent: StripeModel {
     /// Status of this PaymentIntent, one of `requires_payment_method`, `requires_confirmation`, `requires_action`, `processing`, `requires_capture`, `canceled`, or `succeeded`.
     public var status: StripePaymentIntentStatus?
     /// The data with which to automatically create a Transfer when the payment is finalized. See the PaymentIntents Connect usage guide for details.
-    public var transferData: [String: String]?
+    public var transferData: StripePaymentIntentTransferData?
     /// A string that identifies the resulting payment as part of a group. See the PaymentIntents Connect usage guide for details.
     public var transferGroup: String?
 }
 
+public struct StripePaymentIntentTransferData: StripeModel {
+	/// Amount intended to be collected by this PaymentIntent. A positive integer representing how much to charge in the smallest currency unit
+	var destination: String?
```

Comment:
Can we mark these properties as  `public`.

Reply:
Of course